csds-client/client/v2/client_test.go

```diff
@@ -38,7 +31,7 @@ func TestParseNodeMatcherWithFile(t *testing.T) {
 		t.Errorf("Parse NodeMatcher Error: %v", err)
 	}
 
-	if !shouldEqualJSON(t, string(get), want) {
+	if !client.ShouldEqualJSON(t, string(get), want) {
```

Comment:
It is a bit confusing to see imported "client" package in the client package. That's why "clientutil" or similar name would be better for code structure and readability

Reply:
Modified this by importing the package client/util as clientUtil